packages/design-tokens/sass/layout-vars.scss

```diff
@@ -4,29 +4,23 @@ $kz-var-layout-content-max-width: var(
   --kz-var-layout-content-max-width,
   1392px
 );
-$kz-var-layout-content-max-width-id: --kz-var-layout-content-max-width;
 $kz-var-layout-content-side-margin: var(
   --kz-var-layout-content-side-margin,
   72px
 );
-$kz-var-layout-content-side-margin-id: --kz-var-layout-content-side-margin;
 $kz-var-layout-mobile-actions-drawer-height: var(
   --kz-var-layout-mobile-actions-drawer-height,
   60px
 );
-$kz-var-layout-mobile-actions-drawer-height-id: --kz-var-layout-mobile-actions-drawer-height;
 $kz-var-layout-navigation-bar-height: var(
   --kz-var-layout-navigation-bar-height,
   72px
 );
-$kz-var-layout-navigation-bar-height-id: --kz-var-layout-navigation-bar-height;
 $kz-var-layout-breakpoints-medium: var(
   --kz-var-layout-breakpoints-medium,
   768px
 );
-$kz-var-layout-breakpoints-medium-id: --kz-var-layout-breakpoints-medium;
 $kz-var-layout-breakpoints-large: var(
   --kz-var-layout-breakpoints-large,
   1080px
 );
-$kz-var-layout-breakpoints-large-id: --kz-var-layout-breakpoints-large;
```

Comment:
Same with `kz-var-*-id` - they're not useful (in version 2 of DT there is `$kz-var-id-*`)